kfs-ld/src/main/resources/edu/arizona/kfs/module/ld/spring-ld.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This needs to be added to the new kfs-config.properties that was added under UAF-475, which means (unfortunately) this branch needs to pull in the latest from ua-development to do so. Ask Adam how to do this easily, as he has done exactly this recently. Note, org.kuali is already present there, but not this edu.arizona.